src/main/java/ChatbotRan/RanUi.java

```diff
@@ -0,0 +1,146 @@
+package ChatbotRan;
+
+import java.util.ArrayList;
+
+/**
+ * Displays chatbot responses to the CLI.
+ */
+public class RanUi {
+    /**
+     * Prints the startup message.
+     */
+    public void greet() {
+        System.out.println("Hello. I am ");
+        String art = "__________                \n" +
+                "\\______   \\_____    ____  \n" +
+                " |       _/\\__  \\  /    \\ \n" +
+                " |    |   \\ / __ \\|   |  \\\n" +
+                " |____|_  /(____  /___|  /\n" +
+                "        \\/      \\/     \\/ ";
+        System.out.println(art);
+        System.out.println("What would you like to do today?");
+    }
+
+    /**
+     * Prints a line to separate the next command.
+     */
+    public void line() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Prints all tasks in the TaskList.
+     *
+     * @param taskList list of tasks
+     */
+    public void printTasks(TaskList taskList) {
+        int size = taskList.size();
+        if (size == 0) {
+            System.out.println("You haven't got any tasks.");
+        } else {
+            for (int i = 0; i < size; i++) {
+                System.out.println("Task " + (i + 1) + ":" + taskList.get(i));
+            }
+        }
+    }
+
+    /**
+     * Prints a generic error message.
+     */
+    public void unknown() {
+
+        System.out.println("I didn't understand that.");
+    }
+
+    /**
+     * Prints response after adding a Task
+     *
+     * @param task task
+     */
+    public void addTask(Task task) {
+        System.out.println("I've added this task to the list: ");
+        System.out.println(task);
+    }
+
+    void printNumber(int size) {
```

Comment:
Does size mean number of tasks in the list? Didn't seem apparent to me until I read the method implementation. 
```suggestion
    void printNumber(int listSize) {
```